collect_app/src/main/java/org/odk/collect/android/database/instances/DatabaseInstancesRepository.java

```diff
@@ -96,6 +96,11 @@ public Instance getOneByPath(String instancePath) {
         }
     }
 
+    @Override
+    public List<Instance> getAllUnsent() {
```

Comment:
Could we not just use `getAllByStatus` here to avoid adding a new method to the interface that needs maintained/tested?

Reply:
Yes we can do that, done.